docs/es/week03/03-1.md

```diff
@@ -242,22 +241,22 @@ Example: Suppose the input $x$ is one dimensional and has size of 100 and $w$ ha
 | Output size: | $\frac{100 - (5-1)}{1}=96$ | $\frac{100 - (5-1)}{2}=48$ |
 -->
 
-### Variantes para el operador convolucional en DCNNs
+### Variantes para el operador convolucional en DCNN
 
-1. **Zancadas (Striding)**: en lugar de desplazar la ventana una entrada a la vez, se puede realizar con un paso más largo (dos o tres entradas a la vez).
-Ejemplo: Suponiendo que el dato de entrada $x$ es de una dimensión con un tamaño de 100 y $w$ tiene un tamaño de cinco. El tamaño resultante con una zancada de uno o dos está expuesto en la siguiente tabla:
+1. **Zancadas (*Striding*)**: en lugar de desplazar la ventana en $x$ una entrada a la vez, se puede realizar con un paso más grande (por ejemplo, dos o tres entradas a la vez).
+Ejemplo: Supón que la entrada $x$ es unidimensional y tiene un tamaño de 100 y que $w$ tiene un tamaño de cinco. El tamaño de la salida con una zancada (*stride*) de uno o dos se muestra en la siguiente tabla:

```

Comment:
suggestion: "Supón" to "Supongamos" (tense homogeneity)

Reply:
Same as the previous one. The original is in the imperative as is the translation. The suggestion is in the subjunctive mood. If the original had been "Let's suppose...", I would agree with the suggestion.